src/bot.ts

```diff
@@ -44,6 +46,9 @@ export const startBot = async (): Promise<void> => {
     });
     client.on('error', client.logger.error);
     client.on('ready', initReady);
+    client.on('messageCreate', (message: Message) => {
+      if (client.user) initMessageCreate(client, container.logger, message);
```

Comment:
Use curly brackets for the `if` - imo it looks cleaner

Reply:
In actuality, if there are no linter warnings for something along the lines of `client.user` potentially being undefined, then you should be good to remove `if (client.user)`.